Library/Homebrew/rubocops/cask/ast/cask_block.rb

```diff
@@ -51,6 +51,10 @@ def sorted_toplevel_stanzas
           @sorted_toplevel_stanzas ||= sort_stanzas(toplevel_stanzas)
         end
 
+        def stanzaify(nodes)
```

Comment:
Kind of a weird place for this. Also, this is only used in one place, so may be best to inline this.

Additionally, maybe move the `stanza_comments` method into `Stanza`, since it is constructed with the `node` anyways.